src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -550,6 +550,16 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
       FormatArguments="AndroidCreatePackagePerAbi;10"
       Condition=" '$(AndroidCreatePackagePerAbi)' == 'true' "
   />
+  <AndroidWarning Code="XA1037"
+      ResourceName="XA1037"
+      FormatArguments="AndroidUseIntermediateDesignerFile;10"
+      Condition=" '$(AndroidUseIntermediateDesignerFile)' == 'true'"
+  />
+  <AndroidWarning Code="XA1037"
+      ResourceName="XA1037"
+      FormatArguments="AndroidResgenFile;10"
+      Condition=" '$(AndroidResgenFile)' != ''"
+  />
```

Comment:
Is there 1 test left that uses these? Maybe could assert 1 warning and XA1037?

Reply:
That is a good point, do these changes need to be on main also?

Reply:
Yeah, you could target `main` instead and we can cherry-pick to 9.